source/ed/build_helper.cpp

```diff
@@ -48,29 +48,11 @@ nt::MetaVehicleJourney* builder::get_or_create_metavj(const std::string name) {
     return it->second;
 }
 
-static nt::JourneyPattern* get_or_create_journey_pattern(builder& b, const std::string& uri) {
-    auto it = std::find_if(b.data->pt_data->journey_patterns.begin(),
-                        b.data->pt_data->journey_patterns.end(),
-                        [uri](nt::JourneyPattern* jp) {
-                            return jp->uri == uri;
-                        });
-    if (it == b.data->pt_data->journey_patterns.end()) {
-        auto jp = new nt::JourneyPattern;
-        jp->idx = b.data->pt_data->journey_patterns.size();
-        jp->uri = uri;
-        jp->name = uri;
-        b.data->pt_data->journey_patterns.push_back(jp);
-        return jp;
-    }
-    return *it;
-}
-
-
 VJ::VJ(builder & b, const std::string &line_name, const std::string &validity_pattern,
        bool is_frequency,
        const std::string &/*block_id*/, bool wheelchair_boarding, const std::string& uri,
-       std::string meta_vj_name, std::string jp_uri, const std::string& physical_mode): b(b) {
-    // the vj is owned by the jp, so we need to have the jp before everything else
+       std::string meta_vj_name, std::string /*jp_uri*/, const std::string& physical_mode): b(b) {
```

Comment:
remove that `jp_uri` and `block_id` for real... Maybe in another PR, but a JIRA has to be created in that case